packages/react/src/components/Dropdown/Dropdown.stories.js

```diff
@@ -57,7 +57,83 @@ const items = [
   },
 ];
 
-export const Default = (args) => (
+export const Playground = (args) => (
+  <div style={{ width: 400 }}>
+    <Dropdown
+      {...args}
+      id="default"
+      titleText="Dropdown label"
+      helperText="This is some helper text"
+      label="Dropdown menu options"
+      items={items}
+      itemToString={(item) => (item ? item.text : '')}
+    />
+  </div>
+);
+
+Playground.argTypes = {
+  items: {
+    control: false,
+  },
+  initialSelectedItem: {
+    control: false,
+  },
+  itemToElement: {
+    control: false,
+  },
+  className: {
+    control: false,
+  },
+  id: {
+    control: false,
+  },
+  invalid: {
+    control: {
+      type: 'boolean',
+    },
```

Comment:
Need to add `invalidText`, `warnText`, `titleText`, and `label` args. Clicking them in the controls currently crashes the app